apps-rendering/src/components/embedWrapper.tsx

```diff
@@ -356,11 +378,15 @@ const getSourceDetailsForEmbed = (embed: Embed): SourceDetails => {
 			};
 		case EmbedKind.Generic:
 		case EmbedKind.TikTok:
-		case EmbedKind.EmailSignup:
 			return {
 				source: embed.source,
 				sourceDomain: embed.sourceDomain,
 			};
+		case EmbedKind.EmailSignup:
+			return {
+				source: some('Guardian'),
+				sourceDomain: some('www.theguardian.com'),
```

Comment:
I'm guessing you probably want to read these from `embed.source` and `embed.sourceDomain` now, as with the generic case above? They're currently still hardcoded.

Reply:
D'oh, yes definitely